### PR TITLE
[GD-1915] Adding agreements to sign up forms - admin

### PR DESCRIPTION
**Pull request by mkeiser82**

### CODE REVIEW

File: packages/frontend/src/admin-pages/components/nonprofit/settings/manageOrganization/ManageOrganization.vue [hunk 1]
```diff
@@ -209,6 +209,26 @@
...
+                        <div class="c-page-section-header-text">
+                          <h2 class="c-page-section-title">Event Agreements</h2>
+                        </div>
+                      </header>
+                      <div class="c-page-section__main">
+                        <div class="c-form-item c-form-item--checkbox">
+                          <div class="c-form-item__control">
+                            <ul class="c-input-list c-input-list--checkbox">
+                              <li v-for="agreement in agreements">
+                                <input type="checkbox" name="agreementId[]" :id="'agreementId-'+agreement.id" :value="agreement.id" v-model="formData.agreedIds">
```
Comment (dylanmmatthews) [inline packages/frontend/src/admin-pages/components/nonprofit/settings/manageOrganization/ManageOrganization.vue:223]: Thoughts on making this disabled?

File: packages/lambda/src/repositories/nonprofits.js [hunk 2]
```diff
@@ -510,6 +516,28 @@ NonprofitsRepository.prototype.upsert = function (model, data) {
...
+			return allModels.Nonprofit.upsert(params).then(async response => {
+				const updatedModel = response[0];
+
+				if (! data.NonprofitAgreements) {
+					return updatedModel;
+				}
+
+				// set the agreements
+
+				// data.NonprofitAgreements: [{ agreementId: 10, isChecked: true }, { agreementId: 11, isChecked: false }];
```
Comment (dylanmmatthews) [inline packages/lambda/src/repositories/nonprofits.js:530]: should this still be there?

**Review by dylanmmatthews — CHANGES_REQUESTED** [review]
few requests

File: packages/lambda/src/repositories/nonprofits.js (hunk 2 above)
Comment (mkeiser82) [reply packages/lambda/src/repositories/nonprofits.js:530]: Yeah, its meant to be a code comment documenting the shape of the `data.NonprofitAgreements` coming in.

File: packages/frontend/src/admin-pages/components/nonprofit/settings/manageOrganization/ManageOrganization.vue (hunk 1 above)
Comment (mkeiser82) [reply packages/frontend/src/admin-pages/components/nonprofit/settings/manageOrganization/ManageOrganization.vue:223]: Can you clarify?

**Comment by mkeiser82** [comment]
@dylanmmatthews I am going to merge this into the epic branch (feature/GD-1911) and we can resolve any change requests there.

**Pull request merged**